src/README.md

```diff
@@ -0,0 +1,3 @@
+# GOV.UK Frontend
+
+GOV.UK Frontend source code has moved to the [`govuk-frontend` packages directory](../packages/govuk-frontend).
```

Comment:
```suggestion
GOV.UK Frontend source code has moved to the [`govuk-frontend` package directory](../packages/govuk-frontend).
```
Extreme nitpick: `packages` makes it sound like there's a `packages` subfolder somewhere. Singular package makes it sound as though it's the directory of the `govuk-frontend` package.

This is extremely pedantic... :rofl:

Reply:
Ha, yup, I lost a good couple of minutes toying with other options (another one below)

```suggestion
GOV.UK Frontend source code has moved to the [packages `govuk-frontend` directory](../packages/govuk-frontend).
```

I'm going to rebase with https://github.com/alphagov/govuk-frontend/pull/3561 anyway so I'll change it to yours 😊

Reply:
Done ✅